### DISCUSSION TITLE
clamp cityjson object to the ground

### DISCUSSION THREAD

**Discussion by RAFAMATANANTSOA** [cityjson/specs#195] [Q&A]
Hello everyone,
I have a CityJSON file containing buildings that is stored in my 3DCityDB database. When I display the city objects in Cesium by retrieving the geometry objects from the database, the buildings float. I would like to clamp the 3D objects to the ground, which is the ellipsoid WGS84 terrain in Cesium. Are there any tools or methods that can do this?

Thank you.

**Comment by balazsdukai** [comment]
Hi @RAFAMATANANTSOA , there might be a way to fix the floating buildings in the viewer, without changing the data. I suggest that you ask on the Cesium forum https://community.cesium.com/.
